be/Cf.Domain/Exceptions/Messages/DomainErrors.cs

```diff
@@ -24,4 +24,11 @@ public static class Service
         public static MessageInfo PdfNotProvided => new MessageInfo("Could not find any document attached.");
         public static MessageInfo FailedUpload => new MessageInfo("Failed to process the PDF content. Try again.");
     }
+
+    public static class Review
+    {
+        public static MessageInfo NotValidUser => new MessageInfo("Only the advert creator can post a review");
+
+        public static MessageInfo NotFinished => new MessageInfo("Review can not be posted unless the job status is done");
```

Comment:
```suggestion
        public static MessageInfo NotFinished => new MessageInfo("Review can not be posted unless repairs are done");
```

Reply:
https://blog.joetr.com/avoid-nitpicking-in-code-reviews

Reply:
Job status doesn't make sence for normal user

Reply:
these are our internal errors that the user is not going to be able to see

Reply:
they are not really important

Reply:
hmm

Reply:
unless you are providing them in fe, then it makes sense to change them

Reply:
Ok, leave it, since we don't provide them in fe